### PR TITLE
feat: Add cheqd demo and localnet for tests

### PR DESCRIPTION
**Pull request by DaevMithran**
This PR adds the required Readme files and integrates cheqd modules in the demo

Also included a setup-cheqd in ./.github/actions which starts a localnet for the tests to be run

### CODE REVIEW

File: packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts [hunk 1]
```diff
@@ -106,7 +106,11 @@ export class CheqdAnonCredsRegistry implements AnonCredsRegistry {
           schemaId: `${schema.issuerId}/resources/${schemaResource.id}`,
         },
         registrationMetadata: {},
-        schemaMetadata: {},
+        schemaMetadata: {
```
Comment (TimoGlastra) [inline packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts:109]: We don't support cheqd with indy sdk, so this isn't needed i guess?

Comment (DaevMithran) [reply packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts:109]: Anoncreds api is throwing an error if I don't include it
 https://github.com/hyperledger/aries-framework-javascript/blob/main/packages/anoncreds/src/AnonCredsApi.ts#L264

Comment (TimoGlastra) [reply packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts:109]: What is the error it throws? Are you using indy sdk?

Comment (DaevMithran) [reply packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts:109]: Yea I tried using anoncreds-rs module for anoncreds and indysdk for wallet. It panics at that line and returns invalid did:indy did even though I'm using did:cheqd

Comment (TimoGlastra) [reply packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts:109]: In that case it's probably an issue with using AnonCreds RS and Indy SDK together. It's not intended to be used like that, and thus I don't think we should add code to support the flow

Comment (DaevMithran) [reply packages/cheqd/src/anoncreds/services/CheqdAnonCredsRegistry.ts]: Removed that change

**Comment by TimoGlastra** [comment]
@genaris any idea what's going on here?

@ryjones is this related to the different runners?

**Pull request merged**